decorator/capitalize_decorator.rb

```diff
@@ -0,0 +1,17 @@
+require 'decorator'
+
```

Comment:
### Optional
- [ ] For our project and code to be readable, let us kindly change the directory `methods` to `classes` instead. Since the directory contains classes not methods. ###NB: This is just optional

Reply:
I think the files are missing. I will add them all. I have all the changes on my local machine

Reply:
I think the branch you are using is the master branch instead of the decorator branch

Reply:
If you checkout of the master and clone from the decorator branch all the files are complete